modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -23392,6 +23392,7 @@ you-can-customize-this-page=You can customize this page.
 you-can-email-users-that-have-been-manually-added-to-the-database=You can email users that have been manually added to the database. These users should all have {0} as their password. Launching users resets those new users with a random password and notifies them of their new account status on the portal.
 you-can-fill-out-this-form-only-once.-contact-the-owner-of-the-form-if-you-think-this-is-a-mistake=You can fill out this form only once. Contact the owner of the form if you think this is a mistake.
 you-can-get-a-license-directly-from-google=You can get a license directly from Google.
+you-can-manage-comments-in-the-page-builder-and-the-headless-api=You can manage comments in the page builder and the headless API.
```

Comment:
I think we could sort the message:

you-can-manage-comments-in-the-headless-api-and-the-page-builder=You can manage comments in the headless API and the page builder.